cdap-data-fabric/src/test/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetServiceTestBase.java

```diff
@@ -230,7 +228,7 @@ public void onChange(ServiceDiscovered serviceDiscovered) {
   @After
   public void after() throws Exception {
     Services.chainStop(service, opExecutorService, txManager);
-    namespaceClient.delete(Id.Namespace.DEFAULT);
+    namespaceStore.delete(Id.Namespace.DEFAULT);
```

Comment:
Wouldn't it be better to keep `namespaceClient.delete()`? Since it might do more than just deleting from store.